packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -120,7 +120,7 @@ function lookupComponentPair(
   }
 }
 
-const BUILTIN_KEYWORD_HELPERS: Record<string, object> = {
+export const BUILTIN_KEYWORD_HELPERS: Record<string, object> = {
```

Comment:
are these _all_ things ember adds that glimmer doesn't have?

some of these `-`-prexifed ones are weird.
what's resolve? in-el-null? 🙈 are they used?

Reply:
Many of these are produced by AST transformations. I sort of understand how that came about, but it's not even the best way to accomplish the goal. And now that AST transformations have the ability to create imports in their containing files, we should absolutely update this code to do that rather than rely on these rube goldberg shenanigans.